exchanges/exchange.go

```diff
@@ -773,3 +819,58 @@ func (e *Base) GetAPIURLDefault() string {
 func (e *Base) GetAPIURLSecondaryDefault() string {
 	return e.APIUrlSecondaryDefault
 }
+
+// GetWithdrawPermissions will return each of the exchange's compatible withdrawal methods
+func (e *Base) GetWithdrawPermissions() string {
```

Comment:
The string return is useful for humans but for other logic we would require returning the bitmask directly without the need for string parsing. This means we can just check withdraw permissions like this: 

```
wdPerms := exch.GetWithdrawPermissions(); 
if (wdPerms & AutoWithdrawCrypto) { 
    // exchange supports automatic withdrawal of crypto
}
```

Another useful function would be a function which returns a bool based on the supplied bitmask:

```
wdPerms := AutoWithdrawFiat | AutoWithdrawCrypto
if exch.SupportsWithdrawPermissions(wdPerms) {
      // supports auto withdraw of fiat and crypto
}
```

Lastly we'd need an amendment to the exchange interface so we can access these functions from any application using it outside of individual package usage.